app/parse.py

```diff
@@ -8,5 +15,17 @@
     duration: str
 
 
+def parse_single_course(course_soup: BeautifulSoup) -> Course:
+    return Course(
+        name=course_soup.select_one(SELECTOR_NAME).text,
+        short_description=course_soup.select_one(SELECTOR_DESCRIPTION).text,
+        duration=course_soup.select_one(SELECTOR_DURATION).text,
+    )
+
+
 def get_all_courses() -> list[Course]:
-    pass
+    page = requests.get(BASE_URL).content
+    soup = BeautifulSoup(page, "html.parser")
+    course_elements = soup.select(".ProfessionCard_cardWrapper__JQBNJ")
```

Comment:
Ensure that the CSS selector `.ProfessionCard_cardWrapper__JQBNJ` correctly identifies the course elements on the page. If the page structure changes, this selector may need to be updated.